analysisservice/src/main/java/nu/yona/server/AnalysisServiceApplication.java

```diff
@@ -31,6 +31,8 @@
 @EnableCaching
 public class AnalysisServiceApplication
 {
+	@Autowired
+	CacheManager cacheManager;
```

Comment:
Same question here: Is this used? It's not private, so it's possible, but I guess it's not used.
 Add a line note